keras_cv/metrics/object_detection/box_coco_metrics_test.py

```diff
@@ -90,6 +90,21 @@ def test_coco_metric_suite_evaluate_freq(self):
             # passed which actually modifies the final area under curve value.
             self.assertNotEqual(metrics[metric], 0.0)
 
+    def test_coco_metric_suite_force_eval(self):
+        suite = BoxCOCOMetrics(bounding_box_format="xyxy", evaluate_freq=512)
+        y_true, y_pred, categories = load_samples(SAMPLE_FILE)
+
+        suite.update_state(y_true, y_pred)
+        metrics = suite.result()
+        self.assertAllEqual(metrics, {key: 0 for key in golden_metrics})
+
+        suite.update_state(y_true, y_pred)
+        metrics = suite.result(force=True)
+        for metric in metrics:
+            # The metrics do not match golden metrics because two batches were
+            # passed which actually modifies the final area under curve value.
+            self.assertNotEqual(metrics[metric], 0.0)
```

Comment:
Nit (optional): if we expect these to be >0, we could use assertAllGreater and avoid the loop

Reply:
I assumed so, since you're using `assertAllEqual` with a dictionary above. But maybe not 🤷

Reply:
Equal makes sense, because it will compare the keys via iteration.

For _ in dictionary pulls out the keys, so it will run "string" > 0.0

Reply:
Ahh yes, that's goofy. Nevermind this then :)